indico/modules/designer/placeholders.py

```diff
@@ -71,6 +75,21 @@ def render(cls, registration):
         return name
 
 
+class EventLogoPlaceholder(DesignerPlaceholder):
+    group = 'event'
+    name = 'event_logo'
+    description = _("Event Logo")
+
+    @classmethod
+    def render(cls, registration):
```

Comment:
this is an `event`, not a `registration`

Reply:
Yeah, sorry.